docs/bc-registries-audit-notification-workflow.md

```diff
@@ -88,4 +88,19 @@ Specifics regarding the commands referenced in this workflow can be found here:
 
 ## Audit scripts for Aries VCR/OrgBook and BC Registries Issuer
 
-This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
\ No newline at end of file
+This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
+
+## Troubleshoot Failed Credentials Error
+
+If there are companies that dont't get re-processed because the credentials were created in the credential_log table in the event processor database. Follow the steps below to troubleshoot failed credentials error:
+
+Run the `./manage -e prod getPipelineStatus` command and verify the credentials errors. An example regarding this is shown below in the image:
+
+![BC-Registries-Audit-Errors](./images/bc-registries-audit-errors.png)
+
+
+Running the [`requeueFailedCreds` command](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) command will clear and solve the status of Failed Credentials.
+
+Running the [`requeueOrganization` command](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) and [`queueOrganization` command](./bc-registries-agent-configurations-manage-script.md#queueOrganization-command) will resolve the data issue.
```

Comment:
I don't think this paragraph is relevant to this particular example, is it?  If not please delete.